client/src/components/EmojiPicker.js

```diff
@@ -0,0 +1,126 @@
+'use strict';
+
+import React from 'react';
+import ReactEmoji from "react-emoji";
+import { filterEmojis } from "../utils/emojis";
```

Comment:
Incorporating all the feedback here, we can get rid of '../utils/emojis.js' completely.